cfme/tests/infrastructure/test_scvmm_specific.py

```diff
@@ -34,7 +38,8 @@ def test_no_dvd_ruins_refresh(provider, testing_vm_without_dvd):
 
 @pytest.mark.manual
 @pytest.mark.tier(2)
-def test_template_info_scvmm2016():
+@test_requirements.general_ui
```

Comment:
Q: are you sure it should belong to general UI?

Reply:
I'm not sure what else it would be, since this test is to make sure that all the information is displayed properly in the UI. Is there a `test_requirement` you think is more appropriate?

Reply:
I would add provider-specific requirements to #8672 
What do you think?

Reply:
Yep, I pinged Mike and he added them, so this PR is waiting on #8672 to be merged.

Reply:
So there are a few things going on here, and what I'm going to suggest doesn't have to happen in this PR specifically.

We shouldn't have such small test cases, written for specific providers, for something so generic.

We already have a ton of automation around this exact thing - its called 'Relationship Discovery', and you'll find test modules called 'test_relationships'.  These should be generically automated to check all relationship fields in the UI for a given provider, comparing the counts against wrapanapi's queries to the provider directly.

There are already framework components in place for all of this, implemented under the 'validate_stats' method stacks.

If anything this case should be altogether removed, and the SCVMM provider included in test collection/parametrization for the existing automated relationship tests..

Reply:
Ultimately, as far as test metadata goes, I would probably consider this type of test a 'relationships' marker (new in 8672, but old FA that never had requirement marker).  I'd probably give it a casecomponent of 'WebUI' since its also just validating the webUI is displaying things correctly.

Reply:
@mshriver definitely agreed, this PR is more focused on cleaning up and adding metadata to the existing SCVMM tests. 

In the next sprint, I plan to go through each of these test cases and evaluate (1) whether or not it is necessary and (2) whether or not it can be automated. If it is able to be automated then I write a test for it. 

With that said, I'd like to leave these in here for now so that I know what to test for sign-off purposes.